.github/workflows/build.yml

```diff
@@ -93,22 +93,37 @@ jobs:
         path: ./appmock/build/reports/androidTests/connected/**
 
   analyze:
-    name: SonarCloud
+    name: Analyze with SonarCloud
     runs-on: ubuntu-latest
-    env:
-      GH_ACCESS_TOKEN: ${{ secrets.GH_ACTIONS_CI_TOKEN }}
-    needs: test
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0  # Shallow clones should be disabled
-            # for a better relevancy of analysis
+          # for a better relevancy of analysis
+
+      - uses: actions/checkout@v3
+
+      - name: Set up NDK
+        uses: nttld/setup-ndk@v1
+        with:
+          ndk-version: r22b
 
-      - name: Set up JDK 17
+      - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
-          java-version: 17
-          distribution: 'zulu'
+          java-version: '11'
+          distribution: 'adopt'
```

Comment:
Just curious, do we really care about JDK version? @mdrlzy

Reply:
https://docs.sonarcloud.io/appendices/scanner-environment/

Reply:
Not really, only rare deps require latest version

But sonar warns that it is better to use version 17
> ![warning](https://camo.githubusercontent.com/eba2fe52f113f4ea1ac7786a8b52ee9e138df3ac1187abfcbf9422715f4f58a5/68747470733a2f2f736f6e6172736f757263652e6769746875622e696f2f736f6e6172636c6f75642d6769746875622d7374617469632d7265736f75726365732f76322f636f6d6d6f6e2f6d6573736167655f7761726e696e672d313670782e706e67) The version of Java (11.0.20.1) you have used to run this analysis is deprecated and we will stop accepting it soon. Please update to at least Java 17. Read more [here](https://docs.sonarcloud.io/appendices/scanner-environment/)

Reply:
Better to upgrade then.